pipeline_dp/pipeline_backend.py

```diff
@@ -271,7 +273,16 @@ def flatten(self, col1, col2, stage_name: str):
 class SparkRDDBackend(PipelineBackend):
     """Apache Spark RDD adapter."""
 
+    def __init__(self, sc: SparkContext):
+        self._sc = sc
+
     def map(self, rdd, fn, stage_name: str = None):
+        # TODO(make more elegant solution): workaround for public_partitions
+        # It's more convenient to accept them as a list: filtering data based
```

Comment:
Nit: "It's more convenient .." ->
It is beneficial to accept them as in-memory collection for improving performance of filtering. But for applying map, RDD is required.